pkg/ottl/grammar.go

```diff
@@ -444,6 +458,9 @@ func buildLexer() *lexer.StatefulDefinition {
 		{Name: `Equal`, Pattern: `=`},
 		{Name: `LParen`, Pattern: `\(`},
 		{Name: `RParen`, Pattern: `\)`},
+		{Name: `LCParen`, Pattern: `\{`},
+		{Name: `RCParen`, Pattern: `\}`},
```

Comment:
```suggestion
		{Name: `LBrace`, Pattern: `\{`},
		{Name: `RBrace`, Pattern: `\}`},
```
Mostly a nit, but I think this would be the best way to refer to these.